docs/template.md

```diff
@@ -1,7 +1,25 @@
-# qgrid 
+# qgrid template
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="data" >
+        <q-grid:head> 
+            <q-grid:td> 
+                <i title="{{$column.name}}">{{$column.label}}</i>
+            </q-grid:td>
+        </q-grid:head>
+        <q-grid:body>
+            <q-grid:tr>
+            </q-grid:tr>
+            <q-grid:td key="firstName">
+                <b>{{$value}}</b>
+            </q-grid:td>
+            <q-grid:td key="dateBirth">
```

Comment:
I like something like this option:
`<q-grid:td when='$mode === "edit"'>`
It's more flexible and its semantics allows to use expressions, which can be easily combined like:
`<q-grid:td when="$column.isSelected && $row.name == 'John'">`